### PR TITLE
added swimmer parameter doc

### PR DESCRIPTION
**Pull request by jrmlhermitte**
very small detail but figured i'd as I go

### CODE REVIEW

File: databroker/assets/handlers.py [hunk 1]
```diff
@@ -142,6 +142,8 @@ class HDF5DatasetSliceHandler(HandlerBase):
         key of the single HDF5 Dataset used by this Handler
     frame_per_point : integer, optional
         number of frames to return as one datum, default 1
+     swmr : bool, optional
```
Comment (danielballan) [inline databroker/assets/handlers.py]: Is there a stray space before `swmr` here? That will probably break sphinx formatting.

File: databroker/assets/handlers.py [hunk 2]
```diff
@@ -142,6 +142,8 @@ class HDF5DatasetSliceHandler(HandlerBase):
...
+        Open the hdf5 file in SWMR read mode. Only used when mode = 'r'.
```
Comment (danielballan) [inline databroker/assets/handlers.py]: Could you note, `False by default.`?

**Comment by danielballan** [comment]
👍 👍 👍 👍 👍 👍  for correcting "small details" in our docs! Warms my heart.

**Comment by jrmlhermitte** [comment]
cool thanks! :)

**Pull request merged**